GPU/Common/FramebufferCommon.cpp

```diff
@@ -1419,6 +1420,38 @@ void FramebufferManagerCommon::FindTransferFramebuffers(VirtualFramebuffer *&dst
 		}
 	}
 
+	if (!dstBuffer && PSP_CoreParameter().compat.flags().BlockTransferAllowCreateFB) {
+		float renderWidthFactor = renderWidth_ / 480.0f;
+		float renderHeightFactor = renderHeight_ / 272.0f;
+		// A target for the destination is missing - so just create one!
+		// Make sure this one would be found by the algorithm above so we wouldn't
+		// create a new one each frame.
+		VirtualFramebuffer *vfb = new VirtualFramebuffer{};
+		vfb->fbo = nullptr;
+		vfb->fb_address = dstBasePtr;  // NOTE - not necessarily in VRAM!
+		vfb->fb_stride = dstStride;
+		vfb->z_address = 0;
+		vfb->z_stride =0;
+		vfb->width = std::max(dstWidth, dstStride);
+		vfb->height = dstHeight;
+		vfb->newWidth = vfb->width;
+		vfb->newHeight = vfb->height;
+		vfb->lastFrameNewSize = gpuStats.numFlips;
+		vfb->renderWidth = (u16)(vfb->width * renderWidthFactor);
+		vfb->renderHeight = (u16)(vfb->height * renderHeightFactor);
+		vfb->bufferWidth = vfb->width;
+		vfb->bufferHeight = vfb->height;
+		vfb->format = bpp == 4 ? GE_FORMAT_8888 : GE_FORMAT_5551;  // TODO: We don't really know the 16-bit format here.. at all. Can only guess when it gets used later!
+		vfb->drawnFormat = GE_FORMAT_8888;
+		vfb->usageFlags = FB_USAGE_RENDERTARGET;
+		SetColorUpdated(vfb, 0);
+		textureCache_->NotifyFramebuffer(vfb->fb_address, vfb, NOTIFY_FB_CREATED);
+		vfb->fbo = draw_->CreateFramebuffer({ vfb->renderWidth, vfb->renderHeight, 1, 1, true, (Draw::FBColorDepth)vfb->colorDepth });
+		vfbs_.push_back(vfb);
+		dstBuffer = vfb;
+	}
+	dstBuffer->last_frame_used = gpuStats.numFlips;
```

Comment:
At that point, dstBuffer is guaranteed not to be old. Plus, these targets that we don't otherwise render to but texture from do need their last_frame_used updated, otherwise they'll quickly be decimated.

Reply:
When `BlockTransferAllowCreateFB` is false, if`dstBuffer` cannot be null - why do we even need the `!dstBuffer &&` check above, then?

-[Unknown]

Reply:
Um.. right. Oops, sorry was too hasty :)